addon/templates/components/ember-power-select/multiple.hbs

```diff
@@ -8,7 +8,11 @@
     {{#if resultsLength}}
       {{#ember-power-select/options options=results highlighted=_highlighted selection=selection
         highlight=(action "highlight") select=(action "select") dropdown=dropdown as |option|}}
-        {{yield option}}
+        {{#if baseHasBlock}}
+          {{yield option}}
+        {{else}}
+          {{if optionLabelPath (get option optionLabelPath) option}}
```

Comment:
That is an inline if for calling `(get option optionLabelPath)` only if `optionLabelPath` and just render `option` otherwise.

The only way we could save that `if` is if `get` returned the entire object/param on a given pre initialized path. I.e, does `(get option "")` return `option`?